### PR TITLE
Format small

### PR DESCRIPTION
**Pull request by uribgp**
Fixes #{issue number} 

Description: breaking up the large pull request.

I had a question here: /ignite/ignite/contrib/handlers/time_profilers.py

line 764 - line 767
```
        summary_format = "{} took total {}s [min/index: {}, max/index: {}, mean: {}s, std: {}s]"
        for row in results[-2:]:
            row[3] = "{}s/{}".format(*row[3])  # type: ignore[misc]
            row[4] = "{}s/{}".format(*row[4])  # type: ignore[misc]
```

are rows 3 and 4 two items?  So it would be something along the lines of 
`
row[3] = f"{row[3][0]}s/{row[3][1]}"  # type: ignore[misc]
`

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
Thanks for another PR @uribgp ! 

> I had a question here: /ignite/ignite/contrib/handlers/time_profilers.py

I'd say your guess is correct. Let's ask @harsh8398 to confirm.


To fix the CI failure probably you have to update your branch to the master.

**Comment by harsh8398** [comment]
@uribgp Yes, its a tuple with two items.

File: ignite/contrib/handlers/base_logger.py [hunk 1]
```diff
@@ -53,19 +53,17 @@ def __init__(
         if metric_names is not None:
             if not (isinstance(metric_names, list) or (isinstance(metric_names, str) and metric_names == "all")):
                 raise TypeError(
-                    "metric_names should be either a list or equal 'all', " "got {} instead.".format(type(metric_names))
+                    "metric_names should be either a list or equal 'all', " f"got {type(metric_names)} instead."
```
Comment (vfdev-5) [inline ignite/contrib/handlers/base_logger.py:56]: Could you please merge this string like
```
f"metric_names should be either a list or equal 'all', got {type(metric_names)} instead."
```

File: ignite/contrib/handlers/base_logger.py [hunk 2]
```diff
@@ -111,18 +109,18 @@ class BaseWeightsScalarHandler(BaseHandler):
 
     def __init__(self, model: nn.Module, reduction: Callable = torch.norm, tag: Optional[str] = None) -> None:
         if not isinstance(model, torch.nn.Module):
-            raise TypeError("Argument model should be of type torch.nn.Module, " "but given {}".format(type(model)))
+            raise TypeError("Argument model should be of type torch.nn.Module, " f"but given {type(model)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/base_logger.py:112]: same here

File: ignite/contrib/handlers/base_logger.py [hunk 3]
```diff
@@ -111,18 +109,18 @@ class BaseWeightsScalarHandler(BaseHandler):
...
 
         if not callable(reduction):
-            raise TypeError("Argument reduction should be callable, " "but given {}".format(type(reduction)))
+            raise TypeError("Argument reduction should be callable, " f"but given {type(reduction)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/base_logger.py:115]: and here

File: ignite/contrib/handlers/base_logger.py [hunk 4]
```diff
@@ -136,7 +134,7 @@ class BaseWeightsHistHandler(BaseHandler):
 
     def __init__(self, model: nn.Module, tag: Optional[str] = None):
         if not isinstance(model, torch.nn.Module):
-            raise TypeError("Argument model should be of type torch.nn.Module, " "but given {}".format(type(model)))
+            raise TypeError("Argument model should be of type torch.nn.Module, " f"but given {type(model)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/base_logger.py:137]: here too

File: ignite/contrib/handlers/mlflow_logger.py [hunk 5]
```diff
@@ -226,30 +226,30 @@ def __call__(self, engine: Engine, logger: MLflowLogger, event_name: Union[str,
...
             elif isinstance(value, torch.Tensor) and value.ndimension() == 0:
-                rendered_metrics["{} {}".format(self.tag, key)] = value.item()
+                rendered_metrics[f"{self.tag} {key}"] = value.item()
             elif isinstance(value, torch.Tensor) and value.ndimension() == 1:
                 for i, v in enumerate(value):
-                    rendered_metrics["{} {} {}".format(self.tag, key, i)] = v.item()
+                    rendered_metrics[f"{self.tag} {key} {i}"] = v.item()
             else:
-                warnings.warn("MLflowLogger output_handler can not log " "metrics value type {}".format(type(value)))
+                warnings.warn("MLflowLogger output_handler can not log " f"metrics value type {type(value)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/mlflow_logger.py:243]: here too

File: ignite/contrib/handlers/param_scheduler.py [hunk 6]
```diff
@@ -452,31 +446,28 @@ class ConcatScheduler(ParamScheduler):
...
 
         if not isinstance(schedulers, Sequence):
-            raise TypeError("Argument schedulers should be a sequence, but given {}".format(schedulers))
+            raise TypeError(f"Argument schedulers should be a sequence, but given {schedulers}")
 
         if len(schedulers) < 2:
             raise ValueError(
-                "Argument schedulers should be of more than one parameter schedulers, "
-                "but given {}".format(schedulers)
+                "Argument schedulers should be of more than one parameter schedulers, " f"but given {schedulers}"
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:453]: here too

File: ignite/contrib/handlers/param_scheduler.py [hunk 7]
```diff
@@ -452,31 +446,28 @@ class ConcatScheduler(ParamScheduler):
...
-                "given {} and {}".format(len(schedulers), len(durations))
+                "Incorrect number schedulers or duration values, " f"given {len(schedulers)} and {len(durations)}"
             )
 
         for i, scheduler in enumerate(schedulers):
             if not isinstance(scheduler, ParamScheduler) and not isinstance(scheduler, ParamGroupScheduler):
                 raise TypeError(
-                    "Value at index {} of schedulers should be a parameter scheduler, "
-                    "but given {}".format(i, type(scheduler))
+                    f"Value at index {i} of schedulers should be a parameter scheduler, " f"but given {type(scheduler)}"
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:470]: same here

File: ignite/contrib/handlers/param_scheduler.py [hunk 8]
```diff
@@ -926,12 +913,11 @@ def __init__(
...
         if not isinstance(milestones_values, Sequence):
             raise TypeError(
-                "Argument milestones_values should be a list or tuple, but given {}".format(type(milestones_values))
+                f"Argument milestones_values should be a list or tuple, but given {type(milestones_values)}"
             )
         if len(milestones_values) < 1:
             raise ValueError(
-                "Argument milestones_values should be with at least one value, "
-                "but given {}".format(milestones_values)
+                "Argument milestones_values should be with at least one value, " f"but given {milestones_values}"
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:920]: here as well

File: ignite/contrib/handlers/param_scheduler.py [hunk 9]
```diff
@@ -1073,26 +1057,23 @@ def load_state_dict(self, state_dict: Mapping) -> None:
...
+                f"Input state_dict contains {len(sds)} state_dicts of param group schedulers, "
+                f"but {len(self.schedulers)} needed"
             )
 
         for req_n, s, (n, sd) in zip(self.names, self.schedulers, sds):
             if req_n != n:
                 raise ValueError(
-                    "Name of scheduler from input state dict does not correspond to required one,"
-                    " {} vs {}".format(n, req_n)
+                    "Name of scheduler from input state dict does not correspond to required one," f" {n} vs {req_n}"
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:1076]: here as well

File: ignite/contrib/handlers/polyaxon_logger.py [hunk 10]
```diff
@@ -218,21 +218,21 @@ def __call__(self, engine: Engine, logger: PolyaxonLogger, event_name: Union[str
...
             elif isinstance(value, torch.Tensor) and value.ndimension() == 0:
-                rendered_metrics["{}/{}".format(self.tag, key)] = value.item()
+                rendered_metrics[f"{self.tag}/{key}"] = value.item()
             elif isinstance(value, torch.Tensor) and value.ndimension() == 1:
                 for i, v in enumerate(value):
-                    rendered_metrics["{}/{}/{}".format(self.tag, key, i)] = v.item()
+                    rendered_metrics[f"{self.tag}/{key}/{i}"] = v.item()
             else:
-                warnings.warn("PolyaxonLogger output_handler can not log " "metrics value type {}".format(type(value)))
+                warnings.warn("PolyaxonLogger output_handler can not log " f"metrics value type {type(value)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/polyaxon_logger.py:235]: here

File: ignite/contrib/handlers/time_profilers.py [hunk 11]
```diff
@@ -210,7 +210,7 @@ def _as_last_completed(self, engine: Engine) -> None:
 
     def attach(self, engine: Engine) -> None:
         if not isinstance(engine, Engine):
-            raise TypeError("Argument engine should be ignite.engine.Engine, " "but given {}".format(type(engine)))
+            raise TypeError("Argument engine should be ignite.engine.Engine, " f"but given {type(engine)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/time_profilers.py:213]: here

File: ignite/contrib/handlers/tqdm_logger.py [hunk 12]
```diff
@@ -278,10 +276,10 @@ def __call__(self, engine: Engine, logger: ProgressBar, event_name: Union[str, E
                     rendered_metrics[key] = value.item()
                 elif value.ndimension() == 1:
                     for i, v in enumerate(value):
-                        k = "{}_{}".format(key, i)
+                        k = f"{key}_{i}"
                         rendered_metrics[k] = v.item()
                 else:
-                    warnings.warn("ProgressBar can not log " "tensor with {} dimensions".format(value.ndimension()))
+                    warnings.warn("ProgressBar can not log " f"tensor with {value.ndimension()} dimensions")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/tqdm_logger.py:282]: here

File: ignite/contrib/handlers/visdom_logger.py [hunk 13]
```diff
@@ -376,12 +374,12 @@ def __call__(self, engine: Engine, logger: VisdomLogger, event_name: Union[str,
                 keys.append(key)
             elif isinstance(value, torch.Tensor) and value.ndimension() == 1:
                 values = value  # type: ignore[assignment]
-                keys = ["{}/{}".format(key, i) for i in range(len(value))]
+                keys = [f"{key}/{i}" for i in range(len(value))]
             else:
-                warnings.warn("VisdomLogger output_handler can not log " "metrics value type {}".format(type(value)))
+                warnings.warn("VisdomLogger output_handler can not log " f"metrics value type {type(value)}")
```
Comment (vfdev-5) [inline ignite/contrib/handlers/visdom_logger.py:379]: here

**Review by vfdev-5 — COMMENTED** [review]
Thanks for working on the PR @uribgp !
I left comments to merge 2 strings into a single one

File: ignite/contrib/handlers/visdom_logger.py (hunk 13 above)
Comment (uribgp) [reply ignite/contrib/handlers/visdom_logger.py]: Ok, I think I got them all.  From now on I'll start combining those strings when I see them.

File: ignite/contrib/handlers/param_scheduler.py [hunk 14]
```diff
@@ -531,19 +522,17 @@ def load_state_dict(self, state_dict: Mapping) -> None:
...
         if not isinstance(state_dict, Mapping):
-            raise TypeError("Argument state_dict should be a dictionary, but given {}".format(type(state_dict)))
+            raise TypeError(f"Argument state_dict should be a dictionary, but given {type(state_dict)}")
 
         if "schedulers" not in state_dict:
             raise ValueError(
-                "Required state attribute '{}' is absent in provided state_dict '{}'".format(
-                    "schedulers", state_dict.keys()
-                )
+                f"Required state attribute '{'schedulers'}' is absent in provided state_dict '{state_dict.keys()}'"
```
Comment (vfdev-5) [inline ignite/contrib/handlers/param_scheduler.py:529]:
```suggestion
                f"Required state attribute 'schedulers' is absent in provided state_dict '{state_dict.keys()}'"
```

**Comment by vfdev-5** [comment]
@uribgp please hold on on push more file to this branch as I didn't yet merged this PR. Thanks

**Comment by uribgp** [comment]
Oh sorry, I can withdraw those if it's easier.

**Comment by vfdev-5** [comment]
> Oh sorry, I can withdraw those if it's easier.

Thanks ! Yes, it would be easier.

**Comment by uribgp** [comment]
Apologies.  It's reverting locally but not to the upstream branch.  I did a force push and I believe that undid your merge.

**Comment by vfdev-5** [comment]
@uribgp can you cherry-pick this commit : 8a04530 on your branch ?
```
git cherry-pick 8a04530
```

**Comment by uribgp** [comment]
error: commit 8a04530a3b713909d2debde6a54ba6c7bca295f3 is a merge but no -m option was given.

I checked the 4 files from the branch I force pushed and all 4 show the lines from the merge.  The force push is currently only showing reversions to the files I added afterwards.

**Comment by vfdev-5** [comment]
yeah, sorry, this commit is not the one I thought about...

**Comment by vfdev-5** [comment]
Please, put the change from here : https://github.com/pytorch/ignite/pull/1504#discussion_r546093876
and update to master then and it will be good to go

**Review by vfdev-5 — COMMENTED** [review]
Thanks @uribgp !

**Pull request merged**